app/client/src/common/components/Label.tsx

```diff
@@ -6,7 +6,7 @@ export const Label = styled.label`
   font-weight: normal;
   font-size: 0.85em;
   line-height: 12px;
-  color: #c0c5ce;
+  color: ${(props) => props.theme.labelText};
```

Comment:
Could you change the name from `labelText` to something more generic so that it isn't tied to any particular element?